lib/bitcoin/BitcoinClient.ts

```diff
@@ -89,14 +92,44 @@ export default class BitcoinClient {
     const transaction = await this.createTransaction(transactionData, feeInSatoshis);
     const rawTransaction = transaction.serialize();
 
-    const request = {
-      method: 'sendrawtransaction',
-      params: [
-        rawTransaction
-      ]
-    };
+    return this.broadcastTransactionRpc(rawTransaction);
+  }
 
-    return this.rpcCall(request, true);
+  /**
+   * Broadcasts the specified lock transaction.
+   *
+   * @param bitcoinLockTransaction The transaction object.
+   */
+  public async broadcastLockTransaction (bitcoinLockTransaction: BitcoinLockTransactionModel): Promise<string> {
+    if (!(bitcoinLockTransaction.transactionObject instanceof Transaction)) {
+      throw new Error('Transaction property is not of bitcore-lib.Transaction');

```

Comment:
Discussion: should we start considering throwing specific typed errors (e.g. SidetreeError), or their is no point in this layer?

Reply:
This specific instance is being removed in the next step as we want to save the fully serialized txn so this check becomes obsolete.